docs/adr/0016-not-supporting-type-definitions.md

```diff
@@ -0,0 +1,29 @@
+# Do not support type definitions in TOSCA service template
+
+**User Story:** 
+
+Should *TOSCAna* support node type definitions in the TOSCA service template?
+*[decision drivers | forces]* <!-- optional -->
```

Comment:
I would remove this line, since the "question" isnt answered

Reply:
Do you mean line 5 or line 6? Line 6 must get removed definitely.

Reply:
Line 6